api/configparser/configParser.go

```diff
@@ -0,0 +1,14 @@
+package configparser
+
+import "github.com/ElrondNetwork/elrond-go/config"
+
+// CheckEndpoint will return true if a configuration for the given endpoint exists and it is configured to be opened
+func CheckEndpoint(endpointToCheck string, routesConfig config.APIPackageConfig) bool {
```

Comment:
To simplify the config, we should not bloat it with every route. We could just simply define routes that we don't want active, this function (or `IsEndpointActive`) can return true if route is not defined

Reply:
Yes, it would be easier. But that toml file can now be seen as a minimal API routes documentation because all routes are listed there and you'll have a better overview over what to open or not.